assets/aws/files/bin/teleport-generate-config

```diff
@@ -767,6 +782,7 @@ EOF
 elif [[ "${TELEPORT_ROLE}" == "agent" ]]; then
     echo "agent" > ${USE_CONFD_DIR}/role.agent
     cat >${USE_CONFIG_PATH} <<EOF
+tenant_url: https://${IDEMEUM_TENANT}
```

Comment:
If you want to use the idemeum_presets_enabled as part of startup we need to enable that one as well.

We can do later as well

Reply:
Added to review.